ConsoleApp/Program.cs

```diff
@@ -0,0 +1,98 @@
+﻿static bool Compare(string first, string second)
+{
+    if (first.Length != second.Length)
+    {
+        return false;
+    }
+    else if (first.Length == second.Length)
```

Comment:
you do not need condition here because if first branch won't be true that the only scenario left - `first.Length == second.Length`. it can be simplified a much without it. something like
```suggestion
    if (first.Length != second.Length)
    {
        return false;
    }
    
    for (int i = 0; i < first.Length; i++)
        {
            if (first[i] != second[i])
            {
                return false;
            }
        }
        
        return true;
```